msteams-platform/bots/how-to/conversations/command-bot-in-teams.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Notification bot in Teams
+author: surbhigupta
+description: Learn how a command bot works in Teams, and to add command and responses.
+ms.topic: conceptual
+ms.author: v-amprasad
+ms.localizationpriority: high
+---
+
+# Command bot in Teams
+
+Microsoft Teams allows you to automate simple and repetitive tasks in a conversation. You can build a command bot that can respond to simple commands sent in chats with Adaptive Cards. You can create a command bot template in Teams that responds to chat commands by displaying UI using an Adaptive Card. This enables users to send messages in Teams and your app can provide a response as required.
```

Comment:
You can create a command bot template in Teams... -> You can create a command bot template in Teams Toolkit?

Reply:
Updated as suggested.